demos/multi_camera_multi_target_tracking_demo/python/utils/ie_tools.py

```diff
@@ -11,72 +11,69 @@
  limitations under the License.
 """
 
-import os
+import copy
 
-import logging as log
 import numpy as np
-import cv2 as cv
+import cv2
 
 
 class IEModel:
     """Class for inference of models in the Inference Engine format"""
-    def __init__(self, exec_net, inputs_info, input_key, output_key):
-        self.net = exec_net
-        self.inputs_info = inputs_info
-        self.input_key = input_key
-        self.output_key = output_key
+    def __init__(self, core, model_path, device, num_reqs=1, cpu_extension=''):
+        self.load_model(core, model_path, device, num_reqs, cpu_extension)
         self.reqs_ids = []
 
     def _preprocess(self, img):
         _, _, h, w = self.get_input_shape()
-        img = np.expand_dims(cv.resize(img, (w, h)).transpose(2, 0, 1), axis=0)
+        img = np.expand_dims(cv2.resize(img, (w, h)).transpose(2, 0, 1), axis=0)
         return img
 
     def forward(self, img):
         """Performs forward pass of the wrapped IE model"""
-        res = self.net.infer(inputs={self.input_key: self._preprocess(img)})
-        return np.copy(res[self.output_key])
+        self.infer_requests[0].infer({self.input_tensor_name: self._preprocess(img)})
+        return self.infer_requests[0].get_tensor(self.output_tensor_name).data
```

Comment:
or copy.deepcopy
```suggestion
        return self.infer_requests[0].get_tensor(self.output_tensor_name).data[:]
```

Reply:
Yeah, thank you! I'm going to add copying to `get_tensor().data` in all demos. It will eliminate the chance what demos might produce wrong output because of data is overwritten.